_config.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+// Ensure compatibility with PHP 7.2 ("object" is a reserved word),
+// with SilverStripe 3.6 (using Object) and SilverStripe 3.7 (using SS_Object)
+if (!class_exists('SS_Object')) class_alias('Object', 'SS_Object');
```

Comment:
Can you please format this for PSR-2? It's not something that is enforced in SS3 code but is something we try and follow for any new development =)

```php
if (!class_exists('SS_Object')) {
  class_alias('Object', 'SS_Object');
}
```

Reply:
Of course!